experiment/src/org/labkey/experiment/samples/UploadSamplesHelper.java

```diff
@@ -469,11 +469,17 @@ else if (parts[0].equalsIgnoreCase(ExpData.DATA_INPUT_PARENT))
 
                     if (isEmptyParent)
                     {
-                        if (isMerge)
+                        if (isMerge && !isAliquot)
                             parentDataTypesToRemove.add(namePart);
                     }
                     else
                     {
+                        if (isAliquot)
+                        {
+                            String message = parentColName + " is not allowed for aliquots";

```

Comment:
if the parentColName is aliased in the import via a parent alias this might not match up to the user's import name. 
```suggestion
                            String message = "Sample derivation inputs are not allowed for aliquots.";
```

Or maybe a string constant to match with line 396.